### PR TITLE
MM Animated Icon

### PR DESCRIPTION
**Pull request by nekomaster1000**
(If the animation looks bad, change the "interpolate" from "true" to "false")
(I wanted to test it myself but my IDE was pooping itself)

### CODE REVIEW

File: src/main/resources/assets/miningmaster/textures/item/mm_tab_icon.png.mcmeta [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	  3,
+	  4,
+	  5,
+	  6,
+	  7,
+	  8,
+	  9
+    ]
+  }
+}
```
Comment (SwanX1) [inline src/main/resources/assets/miningmaster/textures/item/mm_tab_icon.png.mcmeta:18]: This isn't even a valid .mcmeta file??

Comment (SwanX1) [inline src/main/resources/assets/miningmaster/textures/item/mm_tab_icon.png.mcmeta:18]: This isn't even a valid .mcmeta file??

**Review by SwanX1 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/miningmaster/textures/item/mm_tab_icon.png.mcmeta (hunk 1 above)
Comment (nekomaster1000) [reply src/main/resources/assets/miningmaster/textures/item/mm_tab_icon.png.mcmeta:18]: bro what
I pulled the formatting straight from soul salt in IE 😭

Comment (SwanX1) [reply src/main/resources/assets/miningmaster/textures/item/mm_tab_icon.png.mcmeta:18]: > bro what I pulled the formatting straight from soul salt in IE sob

and the soul salt isn't animated iirc

> The animation is then controlled using a `.mcmeta` file in JSON format with the same name

https://minecraft.fandom.com/wiki/Resource_Pack#Animation

Comment (nekomaster1000) [reply src/main/resources/assets/miningmaster/textures/item/mm_tab_icon.png.mcmeta:18]: it literally is?? it cycles between two shades of blue from an MCmeta file

**Comment by SwanX1** [comment]
This also could've been implemented better - by reusing the actual textures, thus minimizing filesize

**Comment by nekomaster1000** [comment]
> This also could've been implemented better - by reusing the actual textures
I wouldn't know how to do that. If you would like to, be my guest!

**Pull request merged**